af/fastpath/fastpath/core.py

```diff
@@ -1046,8 +1047,12 @@ def msm_processor(queue):
                     matches = match_fingerprints(measurement)
                 else:
                     matches = []
+                confirmed = bool(len(matches))
```

Comment:
IIRC the fingerprints used by the fastpath differ from those inside of the batch pipeline.

Do we maybe want to mention somewhere this and add a TODO to re-align the two?

Reply:
I'll create a task in the project dashboard.

Reply:
Can you link it in here?

Reply:
Update: as discussed, with #278 we'll  keep the row in the fastpath db table and read from there. This make the re-align less urgent.